src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -765,12 +772,21 @@ message TaskQuery {
     // jobGroupDetail - job group details
     // jobGroupSequence - job group sequence
     // jobType - job type (batch or service)
-    // attributes - comma separated job attribute key/value pairs (for example "key1,key2:value2;k3:value3")
+    // attributes - comma separated job attribute key/value pairs. The same key may occur multiple times, with different
+    //              values (any value matches the filter). A value may be omitted, in which case if the key
+    //              occurs only once, only presence of the key is checked, without value comparison (otherwise the value
+    //              is an empty string). Example filters:
+    //              * 'key1' - matches, if the key is present
+    //              * 'key2:value2' - matches if the attributes contain key 'key2' with value 'value2'
+    //              * 'key3,key3:value3a,key3:value3b' - matches if the attributes contain key 'key3' with value '' or 'value3a' or 'value3b'
+    //              All the above can be passed together as 'key1,key2:value2,key3,key3:value3a,key3:value3b'
     // attributes.op - logical 'and' or 'or' operators, which should be applied to multiple attributes specified in the query
     // jobState - job state (one)
     // taskStates - task states (multiple, comma separated). Empty value is the same as no value set.
     // taskStateReasons - reasons associated with task states (multiple, comma separated)
     // needsMigration - if set to true, return only tasks that require migration
+    // skipSystemFailures - a filter for finished tasks only (does not affect non-finished tasks). If set to true,
+    //                      a finished task that failed due to a system error is filtered out.
```

Comment:
What is a system error?

Reply:
Any of the reason codes denoting system error, as defined in `TaskStatus` message above:
```
    //  * 'crashed'              - container crashed due to some internal system error
    //  * 'transientSystemError' - transient error, not agent specific (for example AWS rate limiting)
    //  * 'localSystemError'     - an error scoped to an agent instance on which a container was run.
    //                             The agent should be quarantined or terminated.
    //  * 'unknownSystemError'   - unknown error which cannot be classified either as local/non-local or transient.
    //                             If there are multiple occurences of this error, the agent should be quarantined or terminated.
```

Reply:
I will add a reference to the system error codes.

Reply:
adding where to find the system errors would be useful here